src/featureFlag/helper.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const API_BASE_URL = 'https://api.realdevsquad.com';
+const mockData = [
+  {
+    flagId: 1,
+    title: 'Dark Mode',
+    enabled: true,
+    roles: ['members', 'no_role', 'super_user'],
+    users: {
+      pcBpdEQ5ydRakHHSpY2p: true,
+      quEpdE34ydRakJSpY45p: false,
+    },
+  },
+  {
+    flagId: 2,
+    title: 'Light Mode',
+    enabled: false,
+    roles: [],
+    users: {
+      pcBpdEQ5ydRakHHSpY2p: true,
+      quEpdE34ydRakJSpY45p: true,
+    },
+  },
+];
+
+function customCreateElement({ type = 'div', attribute = {}, text = null }) {
+  const element = document.createElement(type);
+
+  // Sets all the attributes
+  Object.keys(attribute).forEach((key) => {
+    element.setAttribute(key, attribute[key]);
+  });
+
+  if (text) {
+    element.textContent = text;
+  }
+
+  return element;
+}
+
+async function fetchFlags() {
+  // const data = await fetch(API_BASE_URL + "/flag").then(res => res.josn());
```

Comment:
At the end of line, it will `res.json()`

Reply:
Will fix it 👍